### ISSUE TITLE
piqs - standalone package vs what's included in qutip

### ISSUE THREAD

**Issue by petergthatsme** [nathanshammah/piqs#61]
It looks like qutip contains a piqs.py file. Does that include the full functionality of this package? If not, what is the overlap, and which is considered more up-to-date (and/or still developed)? 
thanks.

P.S. also, thanks for writing this - it seems very useful!

**Comment by nathanshammah** [comment]
Dear @petergthatsme, thank you for your message. 

That's correct, PIQS is now integrated in QuTiP, from version 4.3.1. They are both up to date and wit the same content. Updates in the future might affect first the PIQS standalone library. Both options are fine, yet importing the QuTiP module (piqs.qutip) might be safest, as it occurred to a user, on a machine running Ubuntu, that a compiled component issue arose, which was resolved by downloading QuTiP with pip and using piqs.qutip. 

Incidentally, exactly today we updated the comments in the arXiv paper to mention this, https://arxiv.org/abs/1805.05129, as the corresponding paper has been published. 
We will shortly update the online PIQS documentation to make this clearer. Also, more details on the functionalities included in QuTiP will be given in QuTiP’s 4.3.1 documentation, already some examples of use of piqs.qutip in QuTiP are given in the tutorials section under the “Permutational invariant Lindblad dynamics”, http://qutip.org/tutorials.html.
